src/hooks/useCartUI.ts

```diff
@@ -0,0 +1,16 @@
+import { getCartDescription } from '../utils/cartCalculations';
+
+interface UseCartUIProps {
+  itemCount: number;
+  checkedItemsCount: number;
+}
+
+export const useCartUI = ({ itemCount, checkedItemsCount }: UseCartUIProps) => {
+  const descriptionMessage = () => getCartDescription(itemCount);
```

Comment:
함수일 필요가 있을까요?

```suggestion
  const descriptionMessage = getCartDescription(itemCount);
```

값으로 쓰여도 무방해보여요!

Reply:
굳이 함수로 사용하지 않아도 되는 코드였네요. 개선했어요~!

[refactor: as 타입 단언으로 타입 에러 해결 및 불필요 함수를 값으로 변경](https://github.com/woowacourse/react-shopping-cart/pull/405/commits/ab23fc05ac05933763e6edb02eec419c5ab3178a)